Sources/SwiftParser/Expressions.swift

```diff
@@ -15,7 +15,9 @@
 #else
 @_spi(RawSyntax) import SwiftSyntax
 #endif
-
+#if compiler(>=5.8)
+@_spi(RawSyntax) @_spi(ExperimentalLanguageFeatures) internal import SwiftSyntax
```

Comment:
Could you update the import above instead of adding another import to `SwiftSyntax`?